Add map nginx conf multiple cors header (#2672)
GeotrekCE/Geotrek-admin: docs/changelog.rst

2.58.0+dev
-------------------

**Breaking Change**

> - Template nginx.conf.in was changed to work with multiple rando portals (#2670).

Reviewer: ```suggestion
- Template nginx.conf.in has been changed to work with multiple rando portals (#2670).
```

> First, if you changed file `/opt/geotrek-admin/var/conf/nginx.conf.in`, back it up somewhere.

Reviewer: ```suggestion
First, if you have modified the `/opt/geotrek-admin/var/conf/nginx.conf.in` file, save it somewhere.
```

1 - While installing, choose 'Y' to get the new version.

> 2 - Copy your changes from the backed up file to the new version.

Reviewer: ```suggestion
2 - Copy your changes from the saved file to the new version.
```

> 3 - Execute : dpkg-reconfigure geotrek-admin to reapply your customization.

Reviewer: ```suggestion
3 - Execute : `dpkg-reconfigure geotrek-admin` to reapply your customization.
```
